src/server/pom.xml

```diff
@@ -29,7 +29,8 @@
     <packaging>jar</packaging>
 
     <properties>
-        <dropwizard.version>1.3.14</dropwizard.version>
+        <dropwizard.version>2.0.25</dropwizard.version>
+        <jersey.version>2.33</jersey.version>
```

Comment:
Could be upgraded to 2.35.

Reply:
Update: if upgrading to 2.35, it seems the following extra dependency is required:

```
        <dependency>
            <groupId>org.glassfish.jersey.core</groupId>
            <artifactId>jersey-common</artifactId>
            <version>${jersey.version}</version>
        </dependency>
```